diff --git a/browser/components/moz.build b/browser/components/moz.build
index 63b59082d8c8be455530f81ed19d8fac49937200..176e0ca76d86551b1917bc07b2e9714b8f27bc4a 100644
--- a/browser/components/moz.build
+++ b/browser/components/moz.build
@@ -80,6 +80,9 @@ DIRS += [
     "webrtc",
 ]
 
+# Pulse: Custom components
+DIRS += ["autohide", "center", "gradient", "memory", "msidebar", "setup", "shortcuts", "tor", "verticaltabs", "workspace"]
+
 DIRS += ["build"]
 
 if CONFIG["MOZ_SELECTABLE_PROFILES"]:
